src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx

```diff
@@ -341,7 +341,8 @@ export default class ArcanaTracking extends Module {
 		const lookupLog = this.cardStateLog.slice(this.pullIndex + 1)
 		if (lookupLog.length > 0) {
 				lookupLog.forEach(cardState => {
```

Comment:
Not sure, you didn't changed it. But it looks like there are also some indent issues to